src/assets/src/services/api.ts

```diff
@@ -80,14 +80,15 @@ export const getQueue = async (id: number) => {
 }
 
 export const createQueue = async (
-    name: string, allowed_backends: Set<string>, description?: string, hosts?: User[]
+    name: string, allowed_backends: Set<string>, description?: string, physLocation?: string, hosts?: User[]
```

Comment:
For consistency, please make the parameter name `phys_location`. I think it was done like this in past to be consistent with the backend. Same with `updateQueue` below and when constructing the patch object.
```suggestion
    name: string, allowed_backends: Set<string>, description?: string, phys_location?: string, hosts?: User[]
```